manifests/m/MKLabsCoLtd/StarUML/4.0.1/MKLabsCoLtd.StarUML.locale.en-US.yaml

```
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: MKLabsCoLtd.StarUML
+PackageVersion: 4.0.1
+PackageLocale: en-US
+Publisher: MKLabs Co. Ltd.
+PackageName: StarUML 4.0.1

```

```suggestion
Publisher: MKLabs Co.,Ltd.
PackageName: StarUML
```
Needs to match what control panel sees, version number in PackageName is not needed.